src/main/java/com/chargerrobotics/Robot.java

```diff
@@ -115,10 +115,12 @@ public void disabledPeriodic() {}
   /** This function is called once each time the robot enters autonomous mode. */
   @Override
   public void autonomousInit() {
-    robotContainer.setAutonomous();
-    ArduinoSerialReceiver.start();
-    robotContainer.leds.setMode(LEDMode.AUTONOMOUS);
-    LimelightSubsystem.getInstance().setLEDStatus(false);
+    if (RobotBase.isReal()) {
+      robotContainer.setAutonomous();
```

Comment:
Only `LEDSubsystem`-related code was causing issues. There is nothing wrong in `setAutonomous()`. I also removed the scheduling of `alignToTarget()` in `setAutonomous()` because the robot won't need to face the Power Port during the autonomous challenges this year.